### PR TITLE
Add compat data for IDBFactory

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/IDBFactory

### CODE REVIEW

File: api/IDBFactory.json [hunk 1]
```diff
@@ -0,0 +1,600 @@
...
+          },
+          "chrome": [
+            {
+              "version_added": "24"
+            },
+            {
+              "version_added": "23",
+              "version_removed": "57",
+              "prefix": "webkit"
+            }
```
Comment (jpmedley) [inline api/IDBFactory.json]: Copy this information to webview_android and chrome_android.

File: api/IDBFactory.json [hunk 2]
```diff
@@ -0,0 +1,600 @@
...
+            "chrome": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "23",
+                "version_removed": "24",
+                "prefix": "webkit"
+              }
+            ],
```
Comment (jpmedley) [inline api/IDBFactory.json]: Copy this information to webview_android and chrome_android.

File: api/IDBFactory.json [hunk 3]
```diff
@@ -0,0 +1,600 @@
...
+            },
+            "chrome": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "23",
+                "version_removed": "24",
+                "prefix": "webkit"
+              }
```
Comment (jpmedley) [inline api/IDBFactory.json]: Copy this information to webview_android and chrome_android.

File: api/IDBFactory.json [hunk 4]
```diff
@@ -0,0 +1,600 @@
...
```
Comment (jpmedley) [inline api/IDBFactory.json]: Copy this information to webview_android and chrome_android.

File: api/IDBFactory.json (hunk 1 above)
Comment (maboa) [reply api/IDBFactory.json]: 👍

File: api/IDBFactory.json (hunk 2 above)
Comment (maboa) [reply api/IDBFactory.json]: 👍

File: api/IDBFactory.json (hunk 3 above)
Comment (maboa) [reply api/IDBFactory.json]: 👍

File: api/IDBFactory.json (hunk 4 above)
Comment (maboa) [reply api/IDBFactory.json]: 👍

**Comment by maboa** [comment]
That should be updated now :)

**Pull request merged**